libs/scss-foundation/docs/modules/background-hover.md

```diff
@@ -0,0 +1,19 @@
+# Background hover method
+
+Hover utilities for easy definition of regular and hover background states.
+
+## Background hover
+
+@use scss-foundation/src/modules/states/hover;
+
+- Background hover. Method for set the background color of any element for the regular state, and when the element is
```

Comment:
Method for setting the background color...

For future tasks, please use some online spell checkers before writing documentation.